src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
```

Comment:
Please don't add redundant empty lines to your code. Remove the empty line after the class declaration.